test/indexer/test_hca_indexer.py

```diff
@@ -1276,6 +1276,89 @@ def test_multiple_samples(self):
                 for sample in contents['samples']:
                     self.assertIn(sample['entity_type'], sample_entity_types)
 
+    def test_sample_with_no_donor(self):
+        """
+        Index two bundles for the same project, one bundle has a sample that
+        is connected to a donor and the other bundle has a sample that is not.
+        Verify the lack of a donor is represented in the project aggregate.
+        """
+        # Sample (Specimen): 70d2b85a, Donor: b111e5bf
+        bundle_fqid = self.bundle_fqid(uuid='1fd499c5-f397-4bff-9af0-eb42c37d5fbe',
+                                       version='2021-03-18T11:38:49.884Z')
+        self._index_canned_bundle(bundle_fqid)
+        # Sample (Organoid): df23c109, Donor: none
+        bundle_fqid = self.bundle_fqid(uuid='0722b70c-6778-423d-8fe9-869e2a515d35',
+                                       version='2021-03-18T11:38:49.863Z')
+        self._index_canned_bundle(bundle_fqid)
+        donor_b111e5bf = {
+            'document_id': 'b111e5bf-e907-47f9-8eed-75b2ec5536c5',
+            'biomaterial_id': 'Human_62',
+            'biological_sex': 'male',
+            'genus_species': ['Homo sapiens'],
+            'development_stage': 'human adult stage',
+            'diseases': ['normal'],
+            'organism_age': '62 year',
+            'organism_age_value': '62',
+            'organism_age_unit': 'year',
+            'organism_age_range': {
+                'gte': 1955232000.0,
+                'lte': 1955232000.0
+            }
+        }
+        donor_none = {
+            'document_id': None,
+            'biomaterial_id': None,
+            'biological_sex': None,
+            'genus_species': [None],
+            'development_stage': None,
+            'diseases': [None],
+            'organism_age': None,
+            'organism_age_value': None,
+            'organism_age_unit': None
```

Comment:
Why is organism_age_range missing from this literal?

With that exception, it seems like donor_none can be derived as well.

Reply:
`organism_age_range` is only added to donors inner entities if the donor has a `organism_age_in_seconds`. A donor that has no `organism_age` or `organism_age_unit` will have a `organism_age_in_seconds` of `None`, and thus no `organism_age_range`.